.github/workflows/actions.yml

```diff
@@ -0,0 +1,19 @@
+name: actions
+on:
+  push:
+    branches: [ github-actions ]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Permissions for tabularasa.sh
+        run: chmod +x ./tabularasa.sh
+      - name: Workaround for yarn integrity issue
+        run: ./tabularasa.sh
+      - name: Docker Compose Build
```

Comment:
I believe `docker-compose up` will build if there is not a container already built, so we may be able to remove this step.